.gitignore

```diff
@@ -6,4 +6,4 @@ logs/
 target/
 build/
 dist/
-dbt_project.yml
+/dbt_project.yml
```

Comment:
I had to be more specific about the dbt_project.yml files that are ignored, since I need to test package installation, and each package needs a `dbt_project.yml` file